src/sass/_mixins.scss

```diff
@@ -0,0 +1,111 @@
+@mixin dots {
```

Comment:
zamiast mixina może lepiej byłoby zrobić tutaj placeholder? I zamiast `include` używalibyś `extend`? To będzie dużo lepsze rozwiazanie